src/XMakeTasks/CommandLineBuilderExtension.cs

```diff
@@ -14,12 +14,7 @@ namespace Microsoft.Build.Tasks
     /// <summary>
     /// CommandLineBuilder derived class for specialized logic specific to MSBuild tasks
     /// </summary>
-#if WHIDBEY_VISIBILITY
-    internal
-#else
-    public /* because its used by VJ# Vjc task. */
```

Comment:
This is scary.  If that was the only use should we make this internal again?  I guess that might be a breaking change :disappointed: